.github/workflows/macos.yml

```diff
@@ -50,13 +50,16 @@ jobs:
       - name: Install macos dependencies
         if: ${{ matrix.os == 'macos-11' }}
         run: |
-          # See https://github.com/orgs/Homebrew/discussions/4735
-          # and https://github.com/actions/runner-images/issues/8078
-          brew uninstall --ignore-dependencies harfbuzz
-          brew install --build-from-source harfbuzz
-          brew install cmake googletest bash rename libogg libvorbis glew\
-                       openal-soft sdl2 sdl2_image sdl2_ttf freetype harfbuzz  \
+          brew update
+          brew unlink python && brew link --overwrite python
+          brew reinstall harfbuzz
```

Comment:
FYI -- the `macos-11` runner image was last updated yesterday (https://github.com/actions/runner-images/commit/6cc56bac934d65047ef520dce0e8cdfaa20c98c1). Now it should be fine not to reinstall anything.

Reply:
Can I get rid of all those lines?

Reply:
I think it should be fine, yes.